### DISCUSSION TITLE
Clarity regarding billing cycle.

### DISCUSSION THREAD

**Discussion by renuka010** [community/community#121994] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I stopped my copilot subscription some months back. I again subscribed today. But amount is not debited. Next date of debit is with respect to my previous billing cycle. Then how I will be charged for the time till the next billing date? Whether I will be charged for that fixed amount plus my current 20 days charges?

**Comment by Akash1134** [comment, marked as answer]
Hi @renuka010 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by renuka010** [reply]
Sure thanks @Akash1134

**Reply by CepIbp1950** [reply]
The $39 for monthly services for 30 days subscription period?

**Comment by CepIbp1950** [comment]
The $39 for monthly services for 30 days subscription period?
